Docker/README.md

```diff
@@ -32,64 +67,63 @@ Text line 2
 EOF
 ```
 
-This is used to create the file `fileToCreate` and put the following text lines until the EOF keyword.
+This command will create the text file `fileToCreate` and populate it with the lines of text that follow up but not including the EOF keyword.
 
-You can show the content of the created file using: `cat fileToCreate`
+You can display the content of the created file with the command `cat fileToCreate`.
 
-In order to append text to the file, the first `>` can be replaced with `>>`.
+In order to append text to the file, the first `>` can be replaced with `>>`.  **N.B.** If you only have one greater than character '>' the command will overwrite the file rather than appending to the file.
 
-You can edit the files using **vim** or **nano** text editors.
+If you prefer, you can edit the files using **vim** or **nano** text editors.
 
 # Environment setup
 Estimated time: 15 minutes
 
 ## Proxy consideration
 
-This lab is usually configured on our environment that has a direct access to the internet, however if you want to run this lab on your side and behind a corporate proxy, you need to configure your linux distribution and docker to access the internet through this proxy.
+This lab is usually run in our environment that has a direct access to the Internet. If you want to run this lab on your site behind a corporate proxy, you will have to configure your Linux distribution and Docker to access the Internet via your proxy.
 
-The following instruction are available for a RHEL/Centos 7 distribution, because instructions on the docker part are systemd based.
+The following instruction are available for a RHEL/Centos 7 distribution, because instructions on the Docker part are systemd based.
 
 
- 1. Get the proxy ip and port.
- 2. Make sure your host can resolve the proxy address using `nslookup <proxy>`, if not use the proxy ip.
- 3. Configure your linux package manager to go through the proxy by exporting the http_proxy and https_proxy environment variables:
+ 1. Get the proxy IP and port.
+ 2. Make sure your host can resolve the proxy address using `nslookup <proxy>`, if not use the proxy IP.
+ 3. Configure your Linux package manager to go through the proxy by exporting the http_proxy and https_proxy environment variables:
 
 ```
 export http_proxy=http://<proxy name or ip>:<proxy port>
 export https_proxy=http://<proxy name or ip>:<proxy port>
 ```
 
- 4. Configure docker daemon to use the proxy as explained by this document: https://docs.docker.com/engine/admin/systemd/#http-proxy (in short add Environment="HTTP_PROXY=http://proxy.example.com:80/" to your [services] section)
- 5. **Set the proxy in each of your Dockerfiles** using 
+ 4. Configure Docker daemon to use the proxy as explained by this document: https://docs.Docker.com/engine/admin/systemd/#http-proxy (in short add Environment="HTTP_PROXY=http://proxy.example.com:80/" to your [services] section)
```

Comment:
again I want to keep docker.com. You may have done a global search/replace but that's not always appropriate.